index.bs

```diff
@@ -5798,6 +5803,109 @@ map between [=user contexts=] and [=screen orientation override=].
 
 ### Commands ### {#module-emulation-commands}
 
+#### The emulation.setDisableScriptingOverride Command ####  {#command-emulation-setDisableScriptingOverride}
+
+The <dfn export for=commands>emulation.setDisableScriptingOverride</dfn> command
+emulates enabling or disabling JavaScript on web pages.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl" data-cddl-module="remote-cddl">
+      emulation.SetDisableScriptingOverride = (
+        method: "emulation.setDisableScriptingOverride",
+        params: emulation.SetDisableScriptingOverrideParameters
+      )
+
+      emulation.SetDisableScriptingOverrideParameters = {
+        disable: true / null,
```

Comment:
Only taking `true` or `null` is pretty odd. I think `true` / `false` / `null`, where `null` means "revert to browser default settings" makes more sense.

Reply:
I think if you have an untrusted webdriver session connected to your browser there are more significant problems than "maybe it will re-enable scripting".